src/plugins/share/public/components/share_tabs.tsx

```diff
@@ -34,7 +34,10 @@ export const ShareMenuTabs = () => {
 
   tabs.push(linkTab);
 
-  if (shareMenuItems.length > 0) {
+  const enabledItems = shareMenuItems.filter(({ shareMenuItem }) => !shareMenuItem?.disabled);
+
+  // do not show the export tab if the license is disabled
+  if (shareMenuItems.length > 0 && enabledItems.length > 0) {
     tabs.push(exportTab);
   }
```

Comment:
`enabledItems` comes from `shareMenuItems`, so the condition only needs to look at `enabledItems`
```suggestion
  if (enabledItems.length > 0) {
    tabs.push(exportTab);
  }
```